client/views/teams/ConvertToChannelModal/BaseConvertToChannelModal.tsx

```diff
@@ -0,0 +1,82 @@
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import React, { FC, useState, useCallback } from 'react';
+
+import { IRoom } from '../../../../definition/IRoom';
+import FirstStep from './ModalSteps/FirstStep';
+import SecondStep from './ModalSteps/SecondStep';
+
+const STEPS = {
+	LIST_ROOMS: 'LIST_ROOMS',
+	CONFIRM_CONVERT: 'CONFIRM_CONVERT',
+};
+
+type BaseConvertToChannelModalProps = {
+	onClose: () => void;
+	onCancel: () => void;
+	onConfirm: () => Array<IRoom>;
+	currentStep?: string;
+	rooms: Array<IRoom & { isLastOwner?: string }> | undefined;
```

Comment:
Why not?

```suggestion
	rooms?: Array<IRoom & { isLastOwner?: string }>;
```

Why is mandatory to pass an `undefined`?